### ISSUE TITLE
Configuration option missing to deactivate setting the title (e.g. respect TypoScript config.noPageTitle)

### ISSUE THREAD

**Issue by sypets** [mblaschke/TYPO3-metaseo#532]
**MetaSEO version**: 3.0.0

**TYPO3 version**: 8.7.20

**PHP version**: 7.2

**RealUrl version**: 2.40

There is a TypoScript option to completely deactivate setting the title, e.g.:

`config.noPageTitle=2
`
see [config.noPageTitle](https://docs.typo3.org/typo3cms/TyposcriptReference/Setup/Config/Index.html#nopagetitle)

As far as I could find out, metaseo does not respect this setting.

What I am trying to do: I do not want to deactivate setting of the title by metaseo entirely. I have one page, where an extension takes care of setting the title. For this page, I would like to deactivate setting the title by metaseo with config.noPageTitle or in some other way.

**Comment by thomaszbz** [comment]
@sypets Thanks for catching this.

https://docs.typo3.org/typo3cms/extensions/metaseo/Constants/Index.html#pagetitle shows up a lot of ways how to overwrite the title but none of them can be used to just stay silent.

How about the other extension, shouldn't it respect the setting either? This use case could in turn also be in favour of metaseo: There should be a config setting if metaseo should respect the `config.noPageTitle=2` or not. We'd also need that for BC.

There should also be a setting that keeps metaseo silent, irrespective of what is set in `config.noPageTitle`.

**Comment by sypets** [comment]
I agree. The other extension should respect config.noPageTitle=2 too, so it's probably best to find some way to deactivate it per extension (here: metaseo) and per page. 

I don't know what the recommended way to do this is. In any case, I think it would be useful to be able to turn off title generation by metaseo per page. 

As an alternative, you can (and this is what I do now) is to change the already generated title, but I think it is not ideal to generate it twice (generate it by metaseo and generate it and override it by another extension).
